openbas-api/src/main/java/io/openbas/rest/inject/ExerciseInjectApi.java

```diff
@@ -61,14 +65,16 @@ public Iterable<InjectOutput> exerciseInjectsSimple(
   public Iterable<InjectOutput> exerciseInjectsSimple(
       @PathVariable @NotBlank final String exerciseId,
       @RequestBody @Valid final SearchPaginationInput searchPaginationInput) {
+    Map<String, Join<Base, Base>> joinMap = new HashMap<>();
```

Comment:
**question :** Why are we creating this map at this level ? Maybe I'm missing something but this Hashmap is not used in this class and could be created closer to the point where it is used (in FilterUtilsJpa unless I'm mistaken)

Reply:
You're right! I'm going to check the other references too :)